WarshipGame/Assets/Scripts/Gameplay/Ship.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Holds data specifically for the ship like movement speed and the range it can move
+/// Also activates shaders on selection
+/// </summary>
+[SelectionBase]
+public class Ship : MonoBehaviour
+{
+    [Header("Movement")]
+    [Tooltip("The time it takes to rotate the ship when it needs tot turn to a different tile")]
+    [SerializeField] private float RotationTime = 0.3f;
+    [Tooltip("The time it takes the ship to move from tile to tile")]
+    [SerializeField] private float MovementTime = 1f;
+    [Tooltip("The points are the cost of the range. For example if 3 points are set then the ship can move 3 tiles in range")]
+    [SerializeField] private int Points = 3;
```

Comment:
camelCase
Add a whitespace between [SerializeField] and the next [Tooltip] for easier reading